electrum/wallet.py

```diff
@@ -1715,6 +1715,13 @@ def _is_coin_small_and_unconfirmed(self, utxo: PartialTxInput) -> bool:
         In particular, this test triggers for large "dusting transactions"
         that are used for advertising purposes by some entities.
         see #6960
+        To disable this test, set
+        ```
+            "unconf_utxo_freeze_threshold": 0
+        ```
+        in ~/.electrum/config. Make sure to back up your config file first as
+        JSON syntax is somewhat difficult to get right, and the config file
+        will be overwritten with defaults if you get it wrong.
```

Comment:
The advice text here applies to editing the config in general. Should we maybe document `setconfig` and `getconfig` better instead?